walk-in-interview/src/main/webapp/new-job.js

```diff
@@ -4,24 +4,214 @@
  * is submitted.
  */
 
+/**
+ * Titles for all the sections/fields in different languages.
+ */
+const NEW_JOB_PAGE_TITLE = {
```

Comment:
i think you can move all these localized strings into a strings.js if you want to do it per page, you can do `new-job/strings.en.js`

And have something like:
```
const AppStrings = AppStrings || {}
AppStrings['new-job'] = {
   'page-title': 'New Job Post',
    [...]
}
```